alacritty_macos.yml

```diff
@@ -134,6 +134,13 @@ colors:
     cyan:    '0x66cccc'
     white:   '0xdddddd'
 
+  # Indexed Colors (Optional)
+  #
+  # The indexed colors include all colors form 16 to 256.
```

Comment:
s/form/from

Reply:
Thanks for catching this! I wish the compiler for natural languages would be as strict as `rustc`. :smile:

Reply:
That's what us lowly humans are for ;)

Reply:
Guess I'm a JavaScript human. :P